specifications/android.md

```diff
@@ -87,15 +85,9 @@ The distributor MUST send this action to the registered application to confirm t
 * token: the token supplied by the end user application during registration
 * endpoint: the endpoint
 
-### org.unifiedpush.android.connector.REGISTRATION_REFUSED
-
-The distributor MUST send this action to the registered application if the token is already registered for another application or if the distributor refuses for another reason with the 2 following extras:
-* token: the token supplied by the end user application during registration
-* message: this extra MAY be sent to gives an error message
-
 ### org.unifiedpush.android.connector.REGISTRATION_FAILED
 
-The distributor MUST send this action to the registered application if the registration can not be processed but it is not refused (for instance when the distributor is not connected to its provider server) with the 2 following extras:
+The distributor MUST send this action to the registered application if the token is already registered for another application or if the registration can not be processed (for instance when the distributor is not connected to its provider server) with the 2 following extras:
```

Comment:
It might be useful to add something like, 'if a message is sent here, the connector/end-user app SHOULD display this message to the user'.
The same would also apply to dbus

Reply:
Actually, this is most a development best practice since it is not related to the communication between the connector and the distributor